tests/testthat/test-repo.R

```diff
@@ -82,6 +82,28 @@ test_that("can find gitlab url", {
   expect_equal(package_repo(desc, list()), ref)
 })
 
+test_that("GitLab subgroups are properly parsed", {
+  issue_url <- function(text) {
+    package_repo(desc::desc(text = text), list())$url$issue
+  }
+  target <- "https://gitlab.com/salim_b/r/pkgs/pal/issues/"
+  # 1) from URL field, with and without trailing slash
+  expect_equal(issue_url("URL: https://gitlab.com/salim_b/r/pkgs/pal/"),
```

Comment:
Can you please style according to https://style.tidyverse.org/syntax.html#long-lines ?

Reply:
Done.